### ISSUE TITLE
MediaQuery is Applied before the desired target

### ISSUE THREAD

**Issue by tolumide-ng** [brave/brave-browser#25760]
## Description 
While using Brave browser for development, I noticed that styles applied through media query were applied before the target width of the media query.

## Steps to Reproduce
   1. Install Brave browser on your local machine (macOS m1)
   2. Style a  basic web application using CSS with media query (max-width)
   3. `Cmd + Shift + C` and toggle the `mobile icon` to check for mobile responsiveness.

## Actual result:
Styles set through the media query are applied before the target width.

## Expected result:
Styles set through a media query should be set only within/at the expected target

## Reproduces how often: 
Every time, I only started noticing this today but it has been consistent for about an hour.

## Brave version (brave://version info)
[Version 1.44.105 Chromium: 106.0.5249.91 (Official Build) (arm64)](https://brave.com/latest/)

**Comment by jatinmark** [comment]
I would like to work on this. Can you please assign this to me?

**Comment by rebron** [comment]
@tolumide-ng Are you seeing this same behavior in Chrome?

**Comment by tolumide-ng** [comment]
No, I am not seeing the same behavior on either Chrome of Firefox (I tested on both)

**Comment by rebron** [comment]
cc: @pilgrim-brave Possibly related to fingerprinting?
